examples/nodeapi-receivers-get-200.json

```diff
@@ -21,5 +21,31 @@
           "eth1"
         ],
         "device_id": "05017e08-b329-45f9-a566-a3f99cc11e4d"
+    },
+    {
+        "description": "",
+        "format": "urn:x-nmos:format:data",
+        "tags": {},
+        "caps": {
+            "media_types": [
+                "application/json"
+            ],
+            "event_types": [
+                "number/temperature",
```

Comment:
Should be "number/temperature/*"?

Reply:
Yes, quite possibly given the existing definition within IS-07. I'll make that change.